pkg/cableengine/healthchecker/pinger.go

```diff
@@ -34,7 +34,7 @@ const (
 )
```

Comment:
Note that this file is being moved by https://github.com/submariner-io/submariner/pull/3077.